Formula/arangodb.rb

```diff
@@ -23,41 +26,91 @@ class Arangodb < Formula
   end
 
   def install
-    # clang on 10.8 will still try to build against libstdc++,
-    # which fails because it doesn't have the C++0x features
-    # arangodb requires.
     ENV.libcxx
 
-    args = %W[
-      --disable-dependency-tracking
-      --prefix=#{prefix}
-      --disable-relative
-      --datadir=#{share}
-      --localstatedir=#{var}
-    ]
+    if build.head?
+      args = %W[
+        -DVERBOSE=On
+        -DHOMEBREW=On
+        -DUSE_OPTIMIZE_FOR_ARCHITECTURE=Off
+        -DASM_OPTIMIZATIONS=Off
+        -DCMAKE_BUILD_TYPE=RelWithDebInfo
+        -DCMAKE_C_FLAGS=-O3 -fno-omit-frame-pointer
+        -DCMAKE_CXX_FLAGS=-O3 -fno-omit-frame-pointer
+        -DCMAKE_C_COMPILER=clang
+        -DCMAKE_CXX_COMPILER=clang++
+        -DCMAKE_ASM_COMPILER=as
+        -DETCDIR=#{prefix}/etc
+        -DVARDIR=#{var}
+        -DCMAKE_INSTALL_PREFIX:PATH=#{prefix}
+        -DCMAKE_SKIP_RPATH:BOOL=ON
+        ..
+      ]
 
-    args << "--program-suffix=-unstable" if build.head?
+      mkdir "arangodb-build" do
+        system "cmake", *args
+        system "make", "install"
+      end
 
-    if ENV.compiler != :clang
-      ENV.append "LDFLAGS", "-static-libgcc -static-libstdc++"
-    end
+    else
+      # clang on 10.8 will still try to build against libstdc++,
+      # which fails because it doesn't have the C++0x features
+      # arangodb requires.
+      args = %W[
+        --disable-dependency-tracking
+        --prefix=#{prefix}
+        --disable-relative
+        --datadir=#{share}
+        --localstatedir=#{var}
+      ]
 
-    system "./configure", *args
-    system "make", "install"
+      args << "--program-suffix=-unstable" if build.head?
+
+      if ENV.compiler != :clang
+        ENV.append "LDFLAGS", "-static-libgcc -static-libstdc++"
+      end
+
+      system "./configure", *args
+      system "make", "install"
+    end
   end
 
-  def post_install
-    (var/"arangodb").mkpath
-    (var/"log/arangodb").mkpath
+  if build.head?
+    def post_install
+      (var/"arangodb").mkpath
+      (var/"log/arangodb").mkpath
 
-    system "#{sbin}/arangod" + (build.head? ? "-unstable" : ""), "--upgrade", "--log.file", "-"
+      pw = (0...16).map { (65 + rand(26)).chr }.join
+
+      begin
+        system "#{sbin}/arango-init-database", "--database.password", pw
```

Comment:
We typically create databases without passwords, sadly, because I can promise 99% of users won't pay a bit of notice to the `post_install` output and we'll get Issues like _"I can't access my `arangodb` database, what did you do"_ 😕.